.github/workflows/individual-image-scanner-quay.yaml

```diff
@@ -0,0 +1,159 @@
+---
+name: Fetch security scan results from quay
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  scans:
+    runs-on: ubuntu-latest
+    outputs:
+      as-output: ${{ steps.access-setup-scan.outputs.VULNERABILITIES_EXIST }}
+      cs-output: ${{ steps.cluster-setup-scan.outputs.VULNERABILITIES_EXIST }}
+      gd-output: ${{ steps.gateway-deployment-scan.outputs.VULNERABILITIES_EXIST }}
+      kr-output: ${{ steps.kcp-registrar-scan.outputs.VULNERABILITIES_EXIST }}
+      sc-output: ${{ steps.shellcheck-scan.outputs.VULNERABILITIES_EXIST }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            access-setup:
+            - 'images/access-setup/**'
+            cluster-setup:
+              - 'images/cluster-setup/**'
+            gateway-deployment:
+              - 'images/gateway-deployment/**'
+            kcp-registrar:
+              - 'images/kcp-registrar/**'
+            shellcheck:
+              - 'test/images/shellcheck/**'
```

Comment:
Seems like you forgot to scan the vulnerability-scan image itself.

Reply:
yes, I missed that. Added it now.